kpops/utils/environment.py

```diff
@@ -1,36 +1,42 @@
 import os
-import platform
 from collections import UserDict
-from collections.abc import Callable
+from collections.abc import ItemsView, KeysView, MutableMapping, ValuesView
 
 
-class Environment(UserDict):
-    def __init__(self, mapping=None, /, **kwargs) -> None:
-        transformation = Environment.__get_transformation()
-        if mapping is not None:
-            mapping = {transformation(key): value for key, value in mapping.items()}
-        else:
+class Environment(UserDict[str, str]):
+    """Internal environment wrapping OS environment."""
+
+    def __init__(
+        self, mapping: MutableMapping[str, str] | None = None, /, **kwargs
+    ) -> None:
+        self._global = os.environ
```

Comment:
this still freezes the state of the environment, I think. Why don't you directly refer to `os.environ` ?